docs/site/home/index.html

```diff
@@ -173,31 +172,43 @@ <h3>Contributing to Apache TinkerPop</h3>
          <div class="row">
             <div class="col-xs-8">
                TinkerPop is an open source project that <a href="https://tinkerpop.apache.org/docs/current/dev/developer#_contributing">welcomes contributions</a>. There are many ways to get involved:
-               <p/>
+               <p></p>
                <ol>
+
+                  <li>
+                     Join the <a href="https://discord.gg/ndMpKZcBEE" target="_blank">Discord</a> Server.
+                        <ul>
+                           <li>Chat with users and developers to get answere to your questions.</li>
```

Comment:
Typo: "answers"

Reply:
Thanks for catching that, I have updated the spelling in the file